chapters/managementSuperAdmin.md

```diff
@@ -0,0 +1,5 @@
+# Super Admin management
+## Add super admin
+The *Add super admin* button in the menu allows you to invite a super admin to MaaS.
+As illustrated in Figure 4, you will be asked to enter an e-mail (1), and click on the *Send invitation* button (4). After invitation process is finished, the super admin is redirected to the *Show companies* page.
```

Comment:
Manca l'articolo *the* tra *After* e *invitation*. Al posto di *is* non ci va *has*?